From 7765446288f2d63c187d5a86f328c616aa5fcdf0 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@gaudilabs.com>
Date: Thu, 28 Dec 2017 08:06:05 +0200
Subject: [PATCH] Add dl directory

This saves build time and avoid dependency on connectivity to WWW.
It also insures we reproduce same rootfs each time.

NOTE:
This commit is a pure binary files.
GIT cannot have delta petches for binaries.
Since I use frequent rebase operations I put this
commit to be the first commit despite it is not natural.
This way this commit won't change till we really want it to.
---

diff --git a/dl/acl-2.2.52.src.tar.gz b/dl/acl-2.2.52.src.tar.gz
new file mode 100644
index 0000000..84d263b
--- /dev/null
+++ b/dl/acl-2.2.52.src.tar.gz
Binary files differ
diff --git a/dl/attr-2.4.47.src.tar.gz b/dl/attr-2.4.47.src.tar.gz
new file mode 100644
index 0000000..a822f45
--- /dev/null
+++ b/dl/attr-2.4.47.src.tar.gz
Binary files differ
diff --git a/dl/autoconf-2.69.tar.xz b/dl/autoconf-2.69.tar.xz
new file mode 100644
index 0000000..407896a
--- /dev/null
+++ b/dl/autoconf-2.69.tar.xz
Binary files differ
diff --git a/dl/automake-1.15.1.tar.xz b/dl/automake-1.15.1.tar.xz
new file mode 100644
index 0000000..36d5fbb
--- /dev/null
+++ b/dl/automake-1.15.1.tar.xz
Binary files differ
diff --git a/dl/bash-4.4.tar.gz b/dl/bash-4.4.tar.gz
new file mode 100644
index 0000000..a312384
--- /dev/null
+++ b/dl/bash-4.4.tar.gz
Binary files differ
diff --git a/dl/bison-3.0.4.tar.xz b/dl/bison-3.0.4.tar.xz
new file mode 100644
index 0000000..db8c08d
--- /dev/null
+++ b/dl/bison-3.0.4.tar.xz
Binary files differ
diff --git a/dl/busybox-1.27.2.tar.bz2 b/dl/busybox-1.27.2.tar.bz2
new file mode 100644
index 0000000..f8359f0
--- /dev/null
+++ b/dl/busybox-1.27.2.tar.bz2
Binary files differ
diff --git a/dl/dtc-1.4.4.tar.xz b/dl/dtc-1.4.4.tar.xz
new file mode 100644
index 0000000..7071260
--- /dev/null
+++ b/dl/dtc-1.4.4.tar.xz
Binary files differ
diff --git a/dl/fakeroot_1.20.2.orig.tar.bz2 b/dl/fakeroot_1.20.2.orig.tar.bz2
new file mode 100644
index 0000000..c648fac
--- /dev/null
+++ b/dl/fakeroot_1.20.2.orig.tar.bz2
Binary files differ
diff --git a/dl/flex-2.6.4.tar.gz b/dl/flex-2.6.4.tar.gz
new file mode 100644
index 0000000..fd328f9
--- /dev/null
+++ b/dl/flex-2.6.4.tar.gz
Binary files differ
diff --git a/dl/gcc-linaro-6.4.1-2017.08-x86_64_aarch64-linux-gnu.tar.xz b/dl/gcc-linaro-6.4.1-2017.08-x86_64_aarch64-linux-gnu.tar.xz
new file mode 100644
index 0000000..f0fc7c5
--- /dev/null
+++ b/dl/gcc-linaro-6.4.1-2017.08-x86_64_aarch64-linux-gnu.tar.xz
Binary files differ
diff --git a/dl/gettext-0.19.8.1.tar.xz b/dl/gettext-0.19.8.1.tar.xz
new file mode 100644
index 0000000..9f9b075
--- /dev/null
+++ b/dl/gettext-0.19.8.1.tar.xz
Binary files differ
diff --git a/dl/i2c-tools-3.1.2.tar.xz b/dl/i2c-tools-3.1.2.tar.xz
new file mode 100644
index 0000000..7659e01
--- /dev/null
+++ b/dl/i2c-tools-3.1.2.tar.xz
Binary files differ
diff --git a/dl/kmod-24.tar.xz b/dl/kmod-24.tar.xz
new file mode 100644
index 0000000..198bf1f
--- /dev/null
+++ b/dl/kmod-24.tar.xz
Binary files differ
diff --git a/dl/libtool-2.4.6.tar.xz b/dl/libtool-2.4.6.tar.xz
new file mode 100644
index 0000000..8782cd4
--- /dev/null
+++ b/dl/libtool-2.4.6.tar.xz
Binary files differ
diff --git a/dl/libxml2-2.9.5.tar.gz b/dl/libxml2-2.9.5.tar.gz
new file mode 100644
index 0000000..0671b35
--- /dev/null
+++ b/dl/libxml2-2.9.5.tar.gz
Binary files differ
diff --git a/dl/lm-sensors_3.4.0.orig.tar.bz2 b/dl/lm-sensors_3.4.0.orig.tar.bz2
new file mode 100644
index 0000000..a679081
--- /dev/null
+++ b/dl/lm-sensors_3.4.0.orig.tar.bz2
Binary files differ
diff --git a/dl/lzip-1.19.tar.gz b/dl/lzip-1.19.tar.gz
new file mode 100644
index 0000000..f3e219e
--- /dev/null
+++ b/dl/lzip-1.19.tar.gz
Binary files differ
diff --git a/dl/m4-1.4.18.tar.xz b/dl/m4-1.4.18.tar.xz
new file mode 100644
index 0000000..18f019e
--- /dev/null
+++ b/dl/m4-1.4.18.tar.xz
Binary files differ
diff --git a/dl/mtd-utils-2.0.1.tar.bz2 b/dl/mtd-utils-2.0.1.tar.bz2
new file mode 100644
index 0000000..52d6fda
--- /dev/null
+++ b/dl/mtd-utils-2.0.1.tar.bz2
Binary files differ
diff --git a/dl/ncurses-6.0.tar.gz b/dl/ncurses-6.0.tar.gz
new file mode 100644
index 0000000..7951168
--- /dev/null
+++ b/dl/ncurses-6.0.tar.gz
Binary files differ
diff --git a/dl/patchelf-0.9.tar.bz2 b/dl/patchelf-0.9.tar.bz2
new file mode 100644
index 0000000..2f8db21
--- /dev/null
+++ b/dl/patchelf-0.9.tar.bz2
Binary files differ
diff --git a/dl/pkgconf-0.9.12.tar.bz2 b/dl/pkgconf-0.9.12.tar.bz2
new file mode 100644
index 0000000..da1c467
--- /dev/null
+++ b/dl/pkgconf-0.9.12.tar.bz2
Binary files differ
diff --git a/dl/readline-7.0.tar.gz b/dl/readline-7.0.tar.gz
new file mode 100644
index 0000000..c46dd68
--- /dev/null
+++ b/dl/readline-7.0.tar.gz
Binary files differ
diff --git a/dl/u-boot-2017.09.tar.bz2 b/dl/u-boot-2017.09.tar.bz2
new file mode 100644
index 0000000..3eaf490
--- /dev/null
+++ b/dl/u-boot-2017.09.tar.bz2
Binary files differ
